.circleci/config.yml

```diff
@@ -85,3 +76,13 @@ workflows:
             branches:
               only:
                 - master
+  tagged-master:
+    jobs:
+      - container-release:
+          tags:
+            only: /^v([0-9]+)\.([0-9]+)\.([0-9]+)$/
```

Comment:
With this change we won't build any release candidates; is that what we want?

Reply:
Still WIP. I will update the regex to include release candidates.